### PR TITLE
(WIP) Fixing binding issues

### PR DESCRIPTION
**Pull request by vnbaaij**
# Pull Request

## 📖 Description

Several components had missing and/or wrong parameters.
I'm working on the component bindings. This is the status (+ is working, - is not working):
- Select: binding [+] set initial value [-]
- TextField: binding [+] set initial value [+]
- NumberField: binding [+] (1) set initial value [+]
- TextArea: binding [+] set initial value [+]
- Combobox: binding [+] set initial value [+]
- Radiogroup: binding [+] Initial value [+]
- Checkbox: binding [-] Initial value [-]
- Switch: binding [-] Initial value [-]
- Slider: binding [+] Initial value [+]

(1) The binding works in a sense that if you change the value in the non form display, it updates in the non form values overview. It doesn't show the value in the form display BUT when you click on up/down, it does tak the new value as a starting point...
 
### General

- [X] I have added tests for my changes.
- [X] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [X] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [X] I have modified an existing component

## ⏭ Next Steps

Work on the binding issues...

### CODE REVIEW

**Comment by EisenbergEffect** [comment]
@vnbaaij Does this include the work to update to the `current-value` and `current-checked` attributes?

**Comment by vnbaaij** [comment]
@EisenbergEffect Yes it does. Waiting for https://github.com/microsoft/fast/pull/5326 to showup in the Fluentui script so I cando checkbox, etc as well.

**Comment by EisenbergEffect** [comment]
I can help get that out. Let me "make a call" 🤣

File: examples/FluentUIServerSample/Pages/ComponentBindings.razor [hunk 1]
```diff
@@ -147,15 +166,46 @@
...
+    /// </summary>
+    public ComponentBindings()
+    {
+
+        //Form.SelectValue = "Two";
+        //Form.TextFieldValue = "TextFieldValue";
+        //Form.NumberFieldValue = 123;
+        //Form.TextAreaValue = "TextAreaValue";
+        //Form.ComboboxValue = "Help!";
+        Form.SliderValue = 70;
```
Comment (EisenbergEffect) [inline examples/FluentUIServerSample/Pages/ComponentBindings.razor:205]: What's going on with the commented out code here? Is that related to the next set of fixes?

**Review by EisenbergEffect — APPROVED** [review]

File: examples/FluentUIServerSample/Pages/ComponentBindings.razor (hunk 1 above)
Comment (vnbaaij) [reply examples/FluentUIServerSample/Pages/ComponentBindings.razor:205]: Uncommenting allows for testing of the initial value being set for the components

**Comment by EisenbergEffect** [comment]
@nicholasrice Second pair of eyes please 😄

**Comment by vnbaaij** [comment]
I had updated the 'old' ComponentBinding page . Corrected that with the last push.

**Comment by nicholasrice** [comment]
Is this still "WIP"?

**Review by nicholasrice — APPROVED** [review]

**Comment by EisenbergEffect** [comment]
@awentzel Is it ok to merge this even though the static web app CI/CD is failing?

**Comment by awentzel** [comment]
> @awentzel Is it ok to merge this even though the static web app CI/CD is failing?

Yes, we're migrating the Static Web App to FAST hosting and can resolve issues with upcoming PR I'm about to submit.
